packages/manager/modules/octavia-load-balancer/src/create/routing.js

```diff
@@ -37,8 +37,57 @@ export default /* @ngInject */ ($stateProvider) => {
               return filtered;
             }, []),
           ),
+      regionsPlans: /* @ngInject */ ($http, $q, projectId, coreConfig) =>
+        $q
+          .all({
+            plans: $http.get(
+              `/cloud/project/${projectId}/capabilities/productAvailability?addonFamily=octavia-loadbalancer&ovhSubsidiary=${
```

Comment:
**nitpick:** `octavia-loadbalancer` can be maintained as a constant as we may have to use in other  places as well.